settings.gradle.kts

```diff
@@ -113,6 +113,7 @@ dependencyResolutionManagement {
   repositoriesMode = RepositoriesMode.FAIL_ON_PROJECT_REPOS
   repositories {
     mavenCentral()
+    mavenLocal()  // Local builds still take precedence for SNAPSHOT versions
```

Comment:
@gh-yzou this is just done for the purpose of my local testing, will remove once approach looks good to you.